### ISSUE TITLE
should these be removed from here, since these are more of a "nice to have"

### ISSUE THREAD

**Issue by JanneAalto** [frc/wp-essentials-heroku#1]
https://github.com/frc/wp-essentials-heroku/blob/64f5047ab4ca362588a2ad885e3d4881fbc30d21/composer.json#L32-L37

**Comment by jmerilainen** [comment]
Yep maybe. I guess then those would be included to the seed's composer.json or documented clearly? 

On the other hand it is easy to have necessary plugins here to make sure that those are included to projects. For example gravityforms upload plugin is important but easy to forget to add.

**Comment by adeiki** [comment]
But all sites do not have GravityForms. Then of course you could not include these but add those needed separately to project composer.json. Maybe the problem is not in this include  but in not following the best practises of not including unnecessary plugins in the project itself.
